impeller/entity/shaders/geometry/points.comp

```diff
@@ -0,0 +1,67 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <impeller/texture.glsl>
+#include <impeller/types.glsl>
+
+// I dunno.
+layout(local_size_x = 16) in;
+
+layout(std430) readonly buffer PointData {
+  vec2 points[];
```

Comment:
You should include a `uint count` here to guard against OOB access.

Reply:
Oh, I see, you put it as a uniform.

Reply:
Done